### DISCUSSION TITLE
Export pages that use getServerSideProps

### DISCUSSION THREAD

**Discussion by meds** [vercel/next.js#33570] [Ideas]
### Describe the feature you'd like to request

When exporting pages with getServerSideProps fail to export

### Describe the solution you'd like

Either present a warning instead of an error explaining getServersideProps won't function on a statically exported page or have a method to allow exports that ignores getServersideProps.

### Describe alternatives you've considered

getStaticProps doesn't do what I need as I'm looking for data on very dynamic content.

**Comment by leerob** [comment]
To clarify, are you wanting `next export` to be more clear that "⚠️ Detected usage of `getServerSideProps`. This will not work with `next export`. For more information, see the documentation here -> link".

**Reply by meds** [reply]
I want it to make it a warning instead of an error that prevents export.
